src/assisted_test_infra/test_infra/tools/static_network.py

```diff
@@ -134,6 +211,41 @@ def _prepare_interface(logical_interface: str, cidrs: List[str], host_count: int
     return interface
 
 
+def _prepare_bonded_interface(
+    logical_interface: str,
+    starting_physical_interface: int,
+    cidrs: List[str],
+    host_count: int,
+    num_bonded_slaves: int,
+    bonding_mode: str,
+) -> Dict[str, str]:
```

Comment:
```suggestion
) -> Dict[str, Any]:
```
the call to `_prepare_link_aggregation` below stores a dict as a value

Reply:
OK